xbmc/Application.cpp

```diff
@@ -952,11 +933,11 @@ bool CApplication::InitDirectoriesLinux()
     appPath = appBinPath;
     /* Check if binaries and arch independent data files are being kept in
      * separate locations. */
-    if (!CDirectory::Exists(URIUtils::AddFileToFolder(appPath, "language")))
+    if (!CDirectory::Exists(URIUtils::AddFileToFolder(appPath, "system")))
```

Comment:
here. system exists in share as well. Probably my fault, as I remember @Montellese asking about this.